services/simcore/.env

```diff
@@ -58,9 +58,11 @@ STORAGE_ENDPOINT=${PREFIX_STACK_NAME}_storage:8080
 STORAGE_HOST=${PREFIX_STACK_NAME}_storage
 
 SWARM_STACK_NAME=${PREFIX_STACK_NAME}-simcore
+SWARM_STACK_NAME_NO_HYPHEN=${PREFIX_STACK_NAME}_simcore
```

Comment:
what if ``PREFIX_STACK_NAME`` has hyphen? Cannot we use [shell parameter expansions](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) in the makefile for ``PREFIX_STACK_NAME``?

Reply:
yes that is a good point. but normally these are named as master, staging or production